ocs_ci/deployment/deployment.py

```
@@ -2063,8 +2096,16 @@ def deployment_with_ui(self):
-        live_deployment = config.DEPLOYMENT.get("live_deployment")
-        if not live_deployment:
+        fdf_standalone = config.DEPLOYMENT.get("fdf_standalone_deployment", False)
+        if fdf_standalone:
+            # ibm-operators CatalogSource was already created and verified READY
+            # by create_fdf_standalone_catalog_source() before this method was
+            # called — nothing to do here.
+            logger.info(
+                "FDF standalone: ibm-operators CatalogSource already created, "
+                "skipping create_catalog_source()"
+            )
+        elif not live_deployment:
```

Ruff (0.16.1)

[error] Undefined name `live_deployment`

(F821)